sentry/src/main/java/io/sentry/SentryLevel.java

```diff
@@ -18,7 +18,7 @@ public void serialize(final @NotNull ObjectWriter writer, final @NotNull ILogger
     writer.value(name().toLowerCase(Locale.ROOT));
   }
 
-  static final class Deserializer implements JsonDeserializer<SentryLevel> {
+  public static final class Deserializer implements JsonDeserializer<SentryLevel> {
```

Comment:
would it make sense to mark it as `@ApiStatus.Internal` then?

Reply:
hm, we don't really do that for other Deserializers that are public, so I guess it's fine to keep them aligned?